edweiss-app/app.config.ts

```diff
@@ -5,7 +5,6 @@ const config: ExpoConfig = {
 	"name": "EdWeiss",
 	"slug": "edweiss",
 	"version": "1.0.0",
-	"orientation": "portrait",
 	"icon": "./assets/images/icon.png",
```

Comment:
Yes you mentioned this in your PR description. That a great point you made about enabling the screen tilt-rotation. I think you should but it in the wiki in a 'good practice' section, so that if someone what's to disable the screen tilting or enforce the screen position, he can leverage your expertise.

Reply:
I prefer to keep this way as the default for now and ask for other to implement the code I put in the description of this PR if they want to disable it in specific screens. But thank you

Reply:
Yes make more sens. Applying the `least common mechanism` principle 🏗. Makes total sens.